### DISCUSSION TITLE
Add a way to pause all Runelite plugins (zen mode)

### DISCUSSION THREAD

**Discussion by EtienneBoutet** [runelite/runelite#14911] [Ideas]
I think it would be nice to have a way to temporary pause all Runelite plugins with a On­/Off button.

Right now, we have to individually disable plugins if we want zen mode.

**Comment by EliasLahham** [comment]
There's a plugin on the plugin hub called "[Plugin Presets](https://github.com/antero111/plugin-presets)", you could just use that to toggle a "zen" mode

**Reply by EtienneBoutet** [reply]
I didn't knew this plugin existed. Thank you ! :)
